src/lib/components/TextField/TextField.scss

```diff
@@ -100,3 +100,17 @@
 .isRootSizeLarge {
     @include box-field-sizes.size(large);
 }
+
+// Groups
+.isRootGrouped {
```

Comment:
Can @adamkudrna check this? Don't we need to have those values specified by CSS variables? At least the gap?

Reply:
This is being implemented in #454 (and #455).